### ISSUE TITLE
Error when saving expectation suit to disk on Windows OS.

### ISSUE THREAD

**Issue by MrKriss** [fivetran/great_expectations#1483]
**Describe the bug**
The following error occurs when saving expectation suits to disk on Windows.
```
[WinError 123] The filename, directory name, or volume label syntax is incorrect: 'C:\\Users\\Chris Musselle\\projects\\ge_example_project\\great_expectations\\uncommitted/validations/npidata\\warning\\20200523T093819.284016Z\\2020-05-23T09:38:19.284016+00:00'
```
This is possibly due to the colons in the timestamp part of the generated file path not being valid on Windows OS.

**To Reproduce**
1. Follow the `great_expectations init` setup walkthough, fails just after displaying `Generating example Expectation Suite...`
1. Or on running the `context.run_validation_operator()` step in the example notebook created to edit the suite after project setup.  

**Expected behavior**
Successful saving of the expectations suite.

**Environment (please complete the following information):**
 - OS: Windows
 - GE Version: 0.11.0

**Additional context**
From doing a bit of diging it looks like the ISO formatted date string is used as part of the `run_id`, 

https://github.com/great-expectations/great_expectations/blob/3901afe64eca52f3264ba3d49701e774ba28cb81/great_expectations/core/__init__.py#L344-L345

which ultimately becomes part of the a key passed to the `TupleFilesystemStoreBackend`

https://github.com/great-expectations/great_expectations/blob/3901afe64eca52f3264ba3d49701e774ba28cb81/great_expectations/data_context/store/tuple_store_backend.py#L240-L245

**Comment by simon-anders-son** [comment]
Same error here.

**Comment by zack-klein-bx** [comment]
Same error here as well!

**Comment by roblim** [comment]
I have a fix in the works - should be out at the latest, early tomorrow, but likely later today.

**Comment by MrKriss** [comment]
I can confirm all is now fixed as of version 0.11.7. This issue can be closed
